WooCommercePOS/WooCommercePOS/Presentation/PointOfSaleEntryPointView.swift

```diff
@@ -5,7 +5,7 @@ public struct PointOfSaleEntryPointView: View {
     // Temporary. DI proper product models once we have a data layer
     private let viewModel: PointOfSaleDashboardViewModel = {
         let products = ProductFactory.makeFakeProducts()
-        let viewModel = PointOfSaleDashboardViewModel(products: products)
+        let viewModel = PointOfSaleDashboardViewModel(products: products, cardReaderConnectionViewModel: .init(state: .scanningForReader(cancel: {})))
```

Comment:
💯  While simplified at this point, I like that we have some sort of init state which should help with testability if we want to start a flow in a specific stage, something we should think about implementing in the final designs